### PR TITLE
Add SMS OTP Authenticator page

### PR DESCRIPTION
**Pull request by silvashenol20**
This PR is to fix the following issue: [https://github.com/wso2/docs-open-banking/issues/516](https://github.com/wso2/docs-open-banking/issues/516)

### CODE REVIEW

File: en/docs/get-started/configure-sms-otp-authenticator.md [hunk 1]
```diff
@@ -0,0 +1,144 @@
...
+    Log.info("Authentication Successful");
+    publishAuthData(context, "AuthenticationSuccessful", {'psuChannel': psuChannel});
+    OTPFlow(context);
+    },
+    onFail: function (context) {
+    Log.info("Authentication Failed");
+    publishAuthData(context, "AuthenticationFailed", {'psuChannel': psuChannel});
+    }
+    });
+    };
```
Comment (isharailanga) [inline en/docs/get-started/configure-sms-otp-authenticator.md:30]: Shall we format this code. We can use

Comment (isharailanga) [reply en/docs/get-started/configure-sms-otp-authenticator.md:30]: var psuChannel = 'Online Banking';

var onLoginRequest = function(context) {
    publishAuthData(context, "AuthenticationAttempted", {
        'psuChannel': psuChannel
    });
    executeStep(1, {
        onSuccess: function(context) {
            Log.info("Authentication Successful");
            publishAuthData(context, "AuthenticationSuccessful", {
                'psuChannel': psuChannel
            });
            OTPFlow(context);
        },
        onFail: function(context) {
            Log.info("Authentication Failed");
            publishAuthData(context, "AuthenticationFailed", {
                'psuChannel': psuChannel
            });
        }
    });
};

var OTPFlow = function(context) {
    executeStep(2, {
        //OTP-authentication
        onSuccess: function(context) {
            context.selectedAcr = "urn:openbanking:psd2:sca";
            publishAuthData(context, "AuthenticationSuccessful", {
                'psuChannel': psuChannel
            });
        },
        onFail: function(context) {
            publishAuthData(context, "AuthenticationFailed", {
                'psuChannel': psuChannel
            });
            OTPFlow(context);
        }
    });
};

File: en/docs/get-started/configure-sms-otp-authenticator.md [hunk 2]
```diff
@@ -0,0 +1,144 @@
...
+
+### Step 3: Making Mobile Phone a Mandatory Claim
+
+1. Go to the **Main menu** > **Identity** > **Claims** > **List** > **http://wso2.org/claims** > **Mobile** > **Edit**.
+
+    ![select_claims_lists](../assets/img/get-started/quick-start-guide/go-to-claims-lists.png)
+
+    ![edit_mobile_claim](../assets/img/get-started/quick-start-guide/edit-mobile-claim.png)
+
+2. Select the **Required** checkbox.
```
Comment (isharailanga) [inline en/docs/get-started/configure-sms-otp-authenticator.md:132]: add a new step. after 1. 

2. Select `http://wso2.org/claims` from the list.
https://ob.docs.wso2.com/en/latest/assets/img/learn/identifier-first/select-claim-from-list.png

File: en/docs/get-started/configure-sms-otp-authenticator.md [hunk 3]
```diff
@@ -0,0 +1,144 @@
...
+
+2. Select the **Lock User Accounts** checkbox.
+
+    ![maximum_login_attempts](../assets/img/get-started/quick-start-guide/maximum-failed-login-attempts.png)
+
+3. Configure the **Maximum Failed Login Attempts**.
+
+4. Scroll down and click **Update**.
+
+### Step 3: Making Mobile Phone a Mandatory Claim
```
Comment (isharailanga) [inline en/docs/get-started/configure-sms-otp-authenticator.md:124]: For this section, shall we replace the content with https://ob.docs.wso2.com/en/latest/learn/identifier-first/#configure-mobile-as-a-mandatory-claim?

**Review by isharailanga — CHANGES_REQUESTED** [review]
Please do the same changes to the [install-and-setup/configuring-sms-otp-authenticator.md](https://github.com/wso2/docs-ob-toolkit-uk/pull/34/files#diff-0b0045f85a15b5a3875f3c9ed22d0474841730bb6e2bbff9fa5ccddb3f12ec67) as well.

File: en/docs/get-started/configure-sms-otp-authenticator.md (hunk 3 above)
Comment (silvashenol20) [reply en/docs/get-started/configure-sms-otp-authenticator.md]: I addressed the above comment in the last commit. Since i saw it lately ;)

**Review by isharailanga — APPROVED** [review]

**Pull request merged**